crates/libs/core/src/types/runtime/health.rs

```diff
@@ -0,0 +1,72 @@
+use mssf_com::FabricTypes::{FABRIC_HEALTH_INFORMATION, FABRIC_HEALTH_REPORT_SEND_OPTIONS};
+use windows_core::PCWSTR;
+
+use crate::{strings::HSTRINGWrap, types::HealthState, HSTRING};
+
+pub type SequenceNumber = i64;
+
+/// FABRIC_HEALTH_INFORMATION
+#[derive(Debug, Clone)]
+pub struct HealthInformation {
+    pub source_id: HSTRING,
+    pub property: HSTRING,
+    pub time_to_live_seconds: u32,
+    pub state: HealthState,
+    pub description: HSTRING,
+    pub sequence_number: SequenceNumber,
```

Comment:
What is the significance of the sequence number here? Assuming the application populates it to ensure ordering in the event or retries or misordered messages on the network?